[Bug]: create-user doesn't work

### Contact Details

_No response_

### What happened?

The `create-user` script doesn't work - it has two missing dependencies:

- `@librechat/backend/lib/db/migrateDb`
- `@librechat/backend/server/services/auth.service`

The latter appears to have been renamed, while the former doesn't seem to exist anymore.


### Steps to Reproduce

`npm run create-user`


### What browsers are you seeing the problem on?

_No response_

### Relevant log output

```shell
$ npm run create-user

> LibreChat@0.5.8 create-user
> node config/create-user.js

node:internal/modules/cjs/loader:1080
  throw err;
  ^

Error: Cannot find module '@librechat/backend/lib/db/migrateDb'
Require stack:
- /home/mindplay/workspace/LibreChat/config/create-user.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:121:18)
    at Object.<anonymous> (/home/mindplay/workspace/LibreChat/config/create-user.js:2:19)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/mindplay/workspace/LibreChat/config/create-user.js' ]
}

Node.js v18.17.1
```
```


### Screenshots

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/config/create-user.js
+++ b/config/create-user.js
@@ -1,6 +1,5 @@
 const connectDb = require('@librechat/backend/lib/db/connectDb');
-const migrateDb = require('@librechat/backend/lib/db/migrateDb');
-const { registerUser } = require('@librechat/backend/server/services/auth.service');
+const { registerUser } = require('@librechat/backend/server/services/AuthService');
 const { askQuestion, silentExit } = require('./helpers');
 const User = require('@librechat/backend/models/User');
 
@@ -26,7 +25,6 @@ const User = require('@librechat/backend/models/User');
     console.orange('Warming up the engines...');
     await connectDb();
     clearTimeout(timeout);
-    await migrateDb();
   } catch (e) {
     console.error(e);
     silentExit(1);

